mobile-app/app/screens/AppNavigator/screens/Auctions/components/BatchCard.tsx

```diff
@@ -41,29 +41,37 @@ export interface BatchCardProps {
 export function BatchCard(props: BatchCardProps): JSX.Element {
   const navigation = useNavigation<NavigationProp<AuctionsParamList>>();
   const { address } = useWalletContext();
-  const { getVaultsUrl } = useDeFiScanContext();
   const { batch, testID, vault } = props;
-  const LoanIcon = getNativeIcon(batch.loan.displaySymbol);
+  const [progressBarHeight, setProgressBarHeight] = useState<number>(0);
   const blockCount = useSelector((state: RootState) => state.block.count) ?? 0;
   const {
     minNextBidInToken,
     totalCollateralsValueInUSD,
     hasFirstBid,
     minNextBidInUSD,
   } = useAuctionBidValue(batch, vault.liquidationPenalty);
-
+  const { timeRemaining, blocksRemaining, blocksPerAuction } = useAuctionTime(
```

Comment:
can we display seconds too? we have a `secondsToDhmsDisplay` used in future swap last time